packages/storybook/package.json

```diff
@@ -11,13 +11,14 @@
     "@aybolit/white-label": "^0.1.0",
     "@conversionxl/cxl-lumo-styles": "^0.2.2",
     "@conversionxl/cxl-ui": "^0.2.1",
-    "@storybook/addon-knobs": "^5.3.13",
-    "@storybook/addon-viewport": "^5.3.13",
-    "@storybook/addons": "^5.3.13",
-    "@storybook/client-api": "^5.3.13",
-    "@storybook/core": "^5.3.13",
-    "@storybook/polymer": "^5.3.13",
+    "@storybook/addon-knobs": "^6.1.14",
+    "@storybook/addon-viewport": "^6.1.14",
+    "@storybook/addons": "^6.1.14",
+    "@storybook/client-api": "^6.1.14",
+    "@storybook/core": "^6.1.14",
+    "@storybook/polymer": "^5.3.21",
```

Comment:
Curious, why did we have to add @storybook/polymer back? What's the path to dropping it in favor of @storybook/web-components?

Reply:
If it's because of `white-label.stories.js`, I think it's time to drop that package altogether.